lib/config/utils.js

```diff
@@ -20,3 +23,33 @@ exports.toIntegerConfig = function toIntegerConfig (configValue) {
   }
   return configValue
 }
+
+exports.getGitCommit = function getGitCommit (repodir) {
+  if (!fs.existsSync(repodir + '/.git/HEAD')) {
```

Comment:
to get rid of this error or warning, i just need to keep my `.git` directory outside the project directory?
https://stackoverflow.com/a/505517/2413043

Reply:
Yes. You can also do a lot of other things. (as mentioned on Matrix). And as mentioned there, the idea is not to prevent people from breaking AGPL but giving them a heads up, when something seems to be not right.